test/v3/removeLiquidity/removeLiquidity.liquiditybootstrapping.integration.test.ts

```diff
@@ -0,0 +1,406 @@
+// pnpm test -- test/v3/removeLiquidity/removeLiquidity.liquiditybootstrapping.integration.test.ts
+import {
+    Address,
+    createTestClient,
+    http,
+    publicActions,
+    TestActions,
+    walletActions,
+    parseUnits,
+    parseEther,
+    Hex,
+} from 'viem';
+
+import {
+    AddLiquidityKind,
+    Slippage,
+    PoolState,
+    CHAINS,
+    ChainId,
+    AddLiquidityInput,
+    PoolType,
+    Permit2Helper,
+    PermitHelper,
+    PERMIT2,
+    PublicWalletClient,
+    LBPParams,
+    CreatePoolLiquidityBootstrappingInput,
+    InitPool,
+    InitPoolInput,
+    RemoveLiquidityKind,
+    RemoveLiquidityProportionalInput,
+    balancerV3Contracts,
+} from '@/index';
+
+import { doCreatePool, getBalances } from '../../lib/utils/';
+
+import {
+    TOKENS,
+    setTokenBalances,
+    approveSpenderOnTokens,
+    sendTransactionGetBalances,
+    assertInitPool,
+} from '../../lib/utils';
+
+import { ANVIL_NETWORKS, startFork } from '../../anvil/anvil-global-setup';
+import { AddLiquidityV3 } from '../../../src/entities/addLiquidity/addLiquidityV3/index';
+import { RemoveLiquidityV3 } from '../../../src/entities/removeLiquidity/removeLiquidityV3/index';
+
+const protocolVersion = 3;
+const poolType = PoolType.LiquidityBootstrapping;
+const chainId = ChainId.SEPOLIA;
+
+const BAL = TOKENS[chainId].BAL;
+const WETH = TOKENS[chainId].WETH;
+
+describe('add liquidity bootstrapping test', () => {
+    let client: PublicWalletClient & TestActions;
+    let tokens: Address[];
+    let rpcUrl: string;
+    let lbpParams: LBPParams;
+    let createPoolInput: CreatePoolLiquidityBootstrappingInput;
+    let poolAddress: Address;
+    let testAddress: Address;
+    let startTime: bigint;
+    let endTime: bigint;
+
+    beforeAll(async () => {
+        ({ rpcUrl } = await startFork(
+            ANVIL_NETWORKS[ChainId[chainId]],
+            undefined,
+            7831624n,
```

Comment:
Same thing here about the blockNumber.

Reply:
I don't need them. I initially planned to add liq to an already deployed lbp. The test now however creates a lbp instead of using an existing one.